web/client/components/import/style/StylePanel.jsx

```diff
@@ -147,19 +150,26 @@ class StylePanel extends React.Component {
                     {this.state.useDefaultStyle ? null : this.props.stylers[this.getGeomType(this.props.selected)]}
                 </Row>
                 <Row key="options">
-                    <Col xs={2}>
-                        <input aria-label="..." type="checkbox" defaultChecked={this.state.useDefaultStyle} onChange={(e) => { this.setState({ useDefaultStyle: e.target.checked }); }} />
-                    </Col>
-                    <Col style={{ paddingLeft: 0, paddingTop: 1 }} xs={10}>
-                        <label><Message msgId="shapefile.defaultStyle" /></label>
-                    </Col>
-
-                    <Col xs={2}>
-                        <input aria-label="..." type="checkbox" defaultChecked={this.state.zoomOnShapefiles} onChange={(e) => { this.setState({ zoomOnShapefiles: e.target.checked }); }} />
-                    </Col>
-                    <Col style={{ paddingLeft: 0, paddingTop: 1 }} xs={10}>
-                        <label><Message msgId="shapefile.zoom" /></label>
-                    </Col>
+                    {this.props.selected && this.props.selected.name === "Annotations" ?
```

Comment:
i'll leave this comment here for the future.
it would be nice that the annotation UI of the styled panel would come from the Annotation plugin and if that UI decides itself if it needs to be shown or not. This is part of design strategy we will try improve in the future

For the moment this changes are fine